src/specs/data_structures.md

```diff
@@ -377,12 +377,14 @@ Nodes contain a single field:
 |------|---------------------------|-------------|
 | `v`  | [HashDigest](#hashdigest) | Node value. |
 
-The base case (an empty tree) is defined as the [hash](#hashing) of the empty string:
+The base case (an empty tree) is defined as the default value:
 
 ```C++
-node.v = 0xe3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+node.v = 0x0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
The hash of nothing still needs to be defined as the base case, as a user can configure an SMT with 0 leaves, as before. The root of a tree with only default leaves should separately be defined, as here.

Reply:
Ah yes, I see - I'll separate these cases.
Is the implication that the base case is always an SMT of height 0? Is an unmodified SMT with height N not considered to be the base case?

Reply:
I wouldn't consider an empty SMT with more than zero leaves the vase case, I'd consider it like an initial case or default case.